Remotion/Web/Development.WebTesting/WebTestHelper.cs

```diff
@@ -285,7 +286,8 @@ public void OnTearDown (bool hasSucceeded)
         Assertion.IsNotNull(_testName, "'{0}' should be set by the test infrastructure calling '{1}'", nameof(_testName), nameof(OnSetUp));
         var screenshotRecorder = new TestExecutionScreenshotRecorder(_testInfrastructureConfiguration.ScreenshotDirectory);
         screenshotRecorder.CaptureCursor();
-        screenshotRecorder.TakeDesktopScreenshot(_testName);
+        if (!MainBrowserSession.Headless)
```

Comment:
```suggestion
        if (!_mainBrowserSession.Headless)
```

Since you accessed the field earlier and we use fields in this method, too.